storm-core/src/jvm/org/apache/storm/container/ResourceIsolationInterface.java

```diff
@@ -56,4 +64,13 @@
      */
     List<String> getLaunchCommandPrefix(String workerId);
 
+    /**
+     * Get the list of PIDs currently in an isolated container
+     * @param workerId the id of the worker to get these for
+     * @return the set of PIDs, this will be combined with
+     * other ways of getting PIDs. An Empty set or null if
+     * no PIDs are found.
+     * @throws IOException on any error
+     */
```

Comment:
Let's just return a Set and have it be empty if no PIDs are found. Callers then don't have to check both cases.